### PR TITLE
Add Merge UI to V2

### PR DESCRIPTION
**Pull request by jbrunton96**
# Description of Changes
Add UI for Merge into V2.

### CODE REVIEW

File: frontend/src/components/tools/merge/MergeSettings.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Stack, Checkbox } from '@mantine/core';
+import { useTranslation } from 'react-i18next';
+import { MergeParameters } from '../../../hooks/tools/merge/useMergeParameters';
+
+interface MergeSettingsProps {
+  parameters: MergeParameters;
+  onParameterChange: <K extends keyof MergeParameters>(key: K, value: MergeParameters[K]) => void;
+  disabled?: boolean;
```
Comment (ConnorYoh) [inline frontend/src/components/tools/merge/MergeSettings.tsx:9]: when is disabled true?

File: frontend/src/tools/Merge.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      {
+        title: "Settings",
+        isCollapsed: base.settingsCollapsed,
+        onCollapsedClick: base.settingsCollapsed ? base.handleSettingsReset : undefined,
+        tooltip: mergeTips,
+        content: (
+          <MergeSettings
+            parameters={base.params.parameters}
+            onParameterChange={base.params.updateParameter}
+            disabled={base.endpointLoading}
```
Comment (ConnorYoh) [inline frontend/src/tools/Merge.tsx:76]: Don't think we need it anymore

File: testing/test_pdf_1.pdf
Comment (ConnorYoh) [inline testing/test_pdf_1.pdf:1]: There is a test fixtures folder somewhere for dummy files. These need to be kept small. Reusing the same files as much as possible.

File: frontend/src/components/tools/merge/MergeSettings.tsx (hunk 1 above)
Comment (jbrunton96) [reply frontend/src/components/tools/merge/MergeSettings.tsx:9]: It's set to disabled on endpoint loading. All settings components have the same behaviour for this, so I'd rather leave it in for the sake of this PR, but happy to consider removing the disabled properties if you don't think they're necessary anymore.

File: testing/test_pdf_1.pdf
Comment (jbrunton96) [reply testing/test_pdf_1.pdf:1]: Yeah I know there are a couple of test PDFs around the repo but they're all weird looking for testing edge cases of stuff, and as far as I've seen, many of us have had to make our own test PDFs like these (just numbered pages) and they're tiny so I think these will be a good utility for everyone going forward.
